configurations/rolling-upgrade-with-sla.yaml

```diff
@@ -11,4 +11,9 @@ upgrade_sstables: false
 n_loaders: 2
 round_robin: true
 
+user_prefix: 'rolling-upgrade-sla'
+
+authorization_in_upgrade: ""
```

Comment:
Let's have a PR to remove those "features"..

I think @roydahan would agree that we don't need that anymore

Reply:
What does it mean?

Reply:
it's a time bomb amos left us, from the days we started enabling auth in scylla :)

we are breaking authorization in rollbacks, but we just first noticed it